spring-cloud-dataflow-server-core/src/main/java/org/springframework/cloud/dataflow/server/config/features/TaskConfiguration.java

```diff
@@ -126,7 +125,7 @@ public TaskRepositoryInitializer taskRepositoryInitializerForDefaultDB(DataSourc
 		@Bean
 		@ConditionalOnMissingBean
 		public TaskDefinitionRepository taskDefinitionRepository(DataSource dataSource, Server server) throws Exception {
```

Comment:
yeah, sure. Created https://github.com/spring-cloud/spring-cloud-dataflow/issues/675 for this.